### PR TITLE
Add 'no sandbox' argument to Chrome

### PR DESCRIPTION
**Pull request by benbristow**
# Problem

* When running Headless Chrome on Linux systems, it's possible that Chrome won't be able to start because of something to do with sandboxing.

* I've observed this issue on the latest version of CentOS 7 with the latest Kernel.

# Solution

* Pass through the `--no-sandbox` flag to Chrome. This fixes the issue.

### CODE REVIEW

File: lib/browser.js [hunk 1]
```diff
@@ -67,10 +68,11 @@ class HeadlessChrome extends EventEmitter {
       this.chromeInstance = await _launchChrome({
         headless: this.options.headless,
         disableGPU: this.options.disableGPU,
+        noSandbox: this.options.chrome.noSandbox,
         port: this.options.chrome.port,
-        userDataDir: this.options.chrome.userDataDir,
+        userDataDir: this.options.chrome.userDataDir,        
         launchAttempts: this.options.chrome.launchAttempts,
-        handleSIGINT: this.options.chrome.handleSIGINT
+        handleSIGINT: this.options.chrome.handleSIGINT,
```
Comment (gr2m) [inline lib/browser.js]: Could you remove the white space on line 73 and the `,` on line 75?

**Comment by gr2m** [comment]
@benbristow I guess the current test fails on CentOS? Maybe we should add the `--no-sandbox` flag to make it pass?

Also CI currently fails, can you have a look why?

File: lib/browser.js (hunk 1 above)
Comment (benbristow) [inline lib/browser.js]: Good spots. My bad.

**Comment by benbristow** [comment]
@gr2m okay, figured out why.

Apparently Chrome/Lighthouse doesn't like being passed blank flags.

Instead of passing blank flags using a ternary I've changed it to be a dumb 'if true then push to array'.

**Comment by gr2m** [comment]
Thanks for the changes, looks good to me 👍 

Could you please run the current test on CentOS? I would test it myself but I’d don’t have access to a CentOS machine

**Comment by benbristow** [comment]
@gr2m it does work. I've tried it in our production environment.

**Comment by LucianoGanga** [comment]
Merged! I'll be publishing this in NPM soon. Sorry for disappearing these days... 

Thanks @gr2m for approving this changes :) 

Cheers!
Luciano

**Pull request merged**